docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -314,7 +314,7 @@ One of `yamlField` or `regex` is required.
 | Field | Type | Description | Required |
 |-|-|-|-|
 | releaseName | string | The release name of helm deployment. By default, the release name is equal to the application name. | No |
-| valueFiles | []string | List of value files should be loaded. | No |
+| valueFiles | []string | List of value files should be loaded. The paths that can be specified are limited to the inside of the directory where the application configuration file exists. | No |
```

Comment:
```suggestion
| valueFiles | []string | List of value files should be loaded. Only files stored under the application directory are allowed | No |
```

Reply:
This one looks better!
I'll apply this change, thank you :+1: